docs/Advanced-Topics.rst

```diff
@@ -15,22 +15,50 @@ Missing Value Handle
 Categorical Feature Support
 ---------------------------
 
--  LightGBM offers good accuracy with integer-encoded categorical features. LightGBM applies
-   `Fisher (1958) <https://www.tandfonline.com/doi/abs/10.1080/01621459.1958.10501479>`_
-   to find the optimal split over categories as
-   `described here <./Features.rst#optimal-split-for-categorical-features>`_. This often performs better than one-hot encoding.
+-  LightGBM offers good accuracy with integer-encoded categorical features. LightGBM offers the following approaches to deal with categorical features:
+
+   -  Method 1: Applies `Fisher (1958) <https://www.tandfonline.com/doi/abs/10.1080/01621459.1958.10501479>`__ to find the optimal split over categories as `described here <./Features.rst#optimal-split-for-categorical-features>`__.
+
+   -  Method 2: Encoding categorical features into numerical values. We provide two encoding options:
+
+      -  **Target encoding**: encode the categorical feature value by the mean of labels of data with the same feature value in the training set. It is easy to overfit the training data if the encoded value of a training data point uses the label of that training data point itself. So we would randomly divide the training data into folds, and when calculating the target encoding for data in one fold, we only consider data in other folds.
```

Comment:
```suggestion
      -  **Target encoding**: encode the categorical feature value by the mean of labels of data with the same feature value in the training set. It is easy to overfit the training data if the encoded value of a training data point uses the label of that training data point itself. So LightGBM randomly divides the training data into folds, and when calculating the target encoding for data in one fold, only considers data in other folds.
```

I think it's important to remove language like "I" or "we" or "you" from documentation talking about LightGBM's behavior, to avoid users being confused and thinking that this doc means they need to do some preprocessing themselves before passing data to LightGBM.

Reply:
Resolved via fab88db.